string model_name
int32  from_val
int32  until_val
int32  increment
---
string filename